lib/Runtime/Language/InlineCache.cpp

```diff
@@ -697,7 +674,7 @@ namespace Js
 #endif
             if (!PHASE_OFF1(Js::CloneCacheInCollisionPhase))
             {
-                if (!inlineCaches[inlineCacheIndex].IsEmpty() && !inlineCaches[inlineCacheIndex].NeedsToBeRegisteredForStoreFieldInvalidation())
+                if (!inlineCaches[inlineCacheIndex].IsEmpty() && !inlineCaches[inlineCacheIndex].NeedsToBeRegisteredForStoreFieldInvalidation() && GetSize() != 1)
```

Comment:
>1 [](start = 160, length = 1)

is this '1' representing MinPropertyStringInlineCacheSize? If yes, could you use MinPropertyStringInlineCacheSize instead?